baseplate/experiments/__init__.py

```diff
@@ -128,10 +159,11 @@ def variant(self, name, user=None, bucketing_event_override=None,
             If set to None, no override will be applied.  Set to None by
             default.  Note that setting bucketing_event_override to True has no
             effect, it will behave the same as when it is set to None.
-        :param dict extra_event_fields: (Optional) Any extra fields you want to
-            add to the bucketing event.
+        :param dict extra_event_fields: (Deprecated) Any extra fields you want to
+            add to the bucketing event. Not presently used.
```

Comment:
🔕  Since this method accepts arbitrary kwargs, you can just remove this if you're not using this anymore, there might be some reason not to that I'm not familiar with though?